utils/swift_build_support/swift_build_support/cmake.py

```diff
@@ -107,6 +107,11 @@ def common_options(self):
             define("CMAKE_C_COMPILER_LAUNCHER:PATH", toolchain.distcc)
             define("CMAKE_CXX_COMPILER_LAUNCHER:PATH", toolchain.distcc)
 
+        if args.cmake_c_launcher:
```

Comment:
What happens if I enable this and distcc. Maybe distcc should be an option to this? Or should we error if we have both passed in?

Reply:
In the future, I plan on removing distcc flag. I will make it error if both are passed in for now.

Reply:
+1!